ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -86,7 +86,15 @@ function unconvertPointCoordinate(point) {
 }
 
 function offMap([longitude, latitude]) {
-  return longitude < -180 || longitude > 180 || latitude < -90 || latitude > 90
+  const normalizedLongitude =
+    longitude -
+    Math.sign(longitude) * (360 * Math.floor((Math.abs(longitude) + 180) / 360))
+  return (
+    normalizedLongitude < -180 ||
```

Comment:
I think we can update this to drop the longitude checks since we're normalizing / supporting the wrapping now.